.github/workflows/ci.yml

```
@@ -59,37 +62,36 @@ jobs:
-
-    - name: Upload binary
-      uses: actions/upload-artifact@v4
-      with:
-        name: binary-${{ matrix.arch }}
-        path: backend/target/${{ matrix.target }}/release/ha-advanced-automation
+      - uses: actions/checkout@v4
 
+      - name: Install Rust toolchain
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          profile: minimal
```

actionlint (1.7.4)

input "profile" is not defined in action "dtolnay/rust-toolchain@stable". available inputs are "components", "target", "targets", "toolchain"

(action)